### ISSUE TITLE
Feature Request - spatial data for Alaska

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#4425]
@mkoo (or anyone else!) - the WKT-now-geog data for Alaska is busted and eats ``ST_Contains`` - can you get me new spatial data of some form?

**Comment by mkoo** [comment]
is it all the quads for AK?

**Comment by mkoo** [comment]
here's what we posted previously:
https://github.com/BNHM/spatial-layers/tree/master/wkt/US_Alaska
Some or all need to be redone? Do these wkt still work?
Or should I convert to another format?

**Comment by dustymc** [comment]
It was AK itself. I found something, and all spatial data (geog and locality) now passes and is constrained to passing ``st_isvalid``.

I don't actually know if the original WKT was invalid or I messed it up somewhere along the way (probably), but it turns out that gmaps (wkt?? both??) can deal with a LOT of slop, and ``geography`` can't.

We might want to think about trying to find higher-resolution shapes at some point, but "meh" for now.
